core/script_language.h

```diff
@@ -119,7 +119,7 @@ class Script : public Resource {
 	virtual void get_script_method_list(List<MethodInfo> *p_list) const = 0;
 	virtual void get_script_property_list(List<PropertyInfo> *p_list) const = 0;
 
-	virtual int get_member_line(const StringName &p_member) const { return 0; }
+	virtual int get_member_line(const StringName &p_member) const { return -1; }
```

Comment:
I mean it doesn't hurt, but to have it be -1, just unusual to have line 0 be a valid line. Maybe VisualScript should fix that and not the editor plugin.